runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -206,23 +194,9 @@ public final class Engine implements Collector, AutoCloseable
             context,
             config,
             extensions,
-            this::readURL);
-
-        this.configURL = config.configURL();
-        String protocol = configURL.getProtocol();
-        if ("file".equals(protocol) || "jar".equals(protocol))
-        {
-            Function<String, String> watcherReadURL = l -> readURL(configURL, l);
-            this.watcherTask = new FileWatcherTask(manager::reconfigure, watcherReadURL);
-        }
-        else if ("http".equals(protocol) || "https".equals(protocol))
-        {
-            this.watcherTask = new HttpWatcherTask(manager::reconfigure, config.configPollIntervalSeconds());
-        }
-        else
-        {
-            throw new UnsupportedOperationException();
-        }
+            this.configURL,
```

Comment:
We are passing `configURL` to `EngineManager` so it can pass it to `EngineConfigWatcher` constructor so it can watch the config for changes. It seemed legit to do `EngineConfigWatcher` instantiation in `EngineManager`. Pls LMK if you want changes here.